src/plugins/profiles/Virial2DProfile.cpp

```diff
@@ -0,0 +1,61 @@
+//
+// Created by Heier on 07/29/2020.
+//
```

Comment:
is this correct?

Reply:
Yes this particular plugin was created by Michaela. We both use it in our shared project. We decided that I will take care of updating the trunk with any new plugins that are developed during the project :)